### ISSUE TITLE
Quasar language pack

### ISSUE THREAD

**Issue by kingreatwill** [quasarframework/quasar#8442]
```
    framework: {
      config: {},

      // iconSet: 'material-icons', // Quasar icon set
      lang: 'zh-CN', //en-US Quasar language pack

      // For special cases outside of where the auto-import stategy can have an impact
      // (like functional components as one of the examples),
      // you can manually specify Quasar components/directives to be available everywhere:
      //
      // components: [],
      // directives: [],

      // Quasar plugins
      plugins: []
    },
```
![image](https://user-images.githubusercontent.com/13087689/107322181-ee67dc00-6ade-11eb-84b5-03d50cfe963f.png)
![image](https://user-images.githubusercontent.com/13087689/107322216-fc1d6180-6ade-11eb-8a12-3d1776e044eb.png)

**Comment by kingreatwill** [comment]
v2.0.0-beta.3

**Comment by rstoenescu** [comment]
Thanks for reporting!
Fix will be available in beta.4

**Comment by slowaways** [comment]
After updating to Quasar v2 Beta 4:
![image](https://user-images.githubusercontent.com/9668277/107577954-dcb33100-6bd1-11eb-92c4-343999d274c4.png)

I removed the "`node_modules`" and type the "`yarn`" again but without success.

**Comment by rstoenescu** [comment]
You're out of RAM, low on disk space or anything similar. Reboot the machine?

**Comment by slowaways** [comment]
> You're out of RAM, low on disk space or anything similar. Reboot the machine?

No. Yes, I already rebooted my machine and the problem persist.

**Comment by slowaways** [comment]
After downgrade to Quasar v2 Beta 3, compilation is successful:

```
yarn add @quasar/app@3.0.0-beta.3
yarn add quasar@2.0.0-beta.3
```

**Comment by rstoenescu** [comment]
Ah, you're right. Made a typo which ruined everything.
Please upgrade to "@quasar/app" beta.5.

**Comment by diadal** [comment]
[intlify] The message format compilation is not supported in this build. Because message compiler isn't included. You need to pre-compilation all message format. So translate function return 'auth.logout.logout' after upgrading to "@quasar/app" beta.5.

**Comment by rstoenescu** [comment]
Fixed the "message compiler isn't included in this build".
Fix will be available in "@quasar/app" 3.0.0-beta.6

Also fixed the casing in the starter kit.
